mesa/space.py

```diff
@@ -1033,11 +1034,19 @@ def place_agent(self, agent: Agent, node_id: int) -> None:
         self.G.nodes[node_id]["agent"].append(agent)
         agent.pos = node_id
 
-    def get_neighbors(self, node_id: int, include_center: bool = False) -> list[int]:
-        """Get all adjacent nodes"""
-        neighbors = list(self.G.neighbors(node_id))
-        if include_center:
-            neighbors.append(node_id)
+    def get_neighbors(
+        self, node_id: int, include_center: bool = False, radius: int = 1
+    ) -> list[int]:
+        """Get all adjacent nodes within a certain radius"""
+        if radius == 1:
+            neighbors = list(self.G.adj[node_id])
```

Comment:
`G.neighbors` is more readable.

Reply:
yet it's just a wrapper of `G.adj` which is anyway a fundamental function of a nx graph so I'd keep this one

Reply:
They give the same output. For people reading the code, it is easier to understand `G.neighbors` without having to consult NetworkX doc.

Reply:
ok, not a big deal using one or the other, done :-)